app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/ui/theme/Type.kt

```diff
@@ -0,0 +1,16 @@
+package au.com.gridstone.trainingkotlin.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+// Set of Material typography styles to start with
+val Typography = Typography(
+  body1 = TextStyle(
+    fontFamily = FontFamily.Default,
+    fontWeight = FontWeight.Normal,
+    fontSize = 16.sp
+  )
+)
```

Comment:
if they are the default, you can still reference `MaterialTheme.typography`

Reply:
sure